pages/blog/json-schema-beyond-validation.md

```diff
@@ -0,0 +1,152 @@
+---
+title: "Using JSON Schema beyond validation"
```

Comment:
we are not very much consistent with cases in titles, but I think the correct one would be to use upper case

```suggestion
title: "Using JSON Schema Beyond Validation"
```